pages/dkp/kaptain/2.0.0/install/index.md

```diff
@@ -20,41 +22,31 @@ You can install Kaptain:
 
 The infrastructure you select determines the specific requirements for a successful installation.
 
-## Before you begin
-
-Kaptain is a DKP Catalog application. To use it, add it to your repository and then deploy it on selected workspaces. No downloads are necessary for networked environments. For users deploying to air-gapped environments, refer to the [download][download] page.
-
-You can deploy Kaptain to single and multi-cluster environments. The difference between these setups is the following: In a single-cluster environment (with an Essential license), you deploy on one Management cluster only. In a multi-cluster environment (with an Enterprise license), Kaptain is deployed to either one or several Managed clusters or to Attached clusters.
-
-You can deploy Kaptain on a per-workspace basis.
-
-<p class="message--warning"><strong>WARNING: </strong>You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.</p>
+## Installation overview
 
-## Prerequisites
+To install and deploy Kaptain for the first time, proceed with these steps:
 
-- A DKP cluster with the following Platform applications enabled:
+1.  Ensure you have reviewed all [prerequisites](../install/prerequisites/).
```

Comment:
do we also want to make a comment about and add the link to "prerequisites for on-premise environments"?